tasks/create_storage_domain.yml

```diff
@@ -1,6 +1,11 @@
 ---
 - name: Create hosted engine local vm
   block:
+  - name: Wait until all interfaces wake up
+    wait_for:
```

Comment:
please provide  more solid condition for the wait_for, this will just cause more delay to all the deployments.